ServiceProviderBot/Bot/Dialogs/UpdateOrganization/Capacity/UpdateCaseManagementDialog.cs

```diff
@@ -0,0 +1,31 @@
+﻿using EntityModel;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Extensions.Configuration;
+using Shared;
+using Shared.ApiInterface;
+using System.Collections.Generic;
+
+namespace ServiceProviderBot.Bot.Dialogs.UpdateOrganization.Capacity
+{
+    public class UpdateCaseManagementDialog : DialogBase
+    {
+        public static string Name = typeof(UpdateCaseManagementDialog).FullName;
+
+        public UpdateCaseManagementDialog(StateAccessors state, DialogSet dialogs, IApiInterface api, IConfiguration configuration)
+            : base(state, dialogs, api, configuration) { }
+
+        public override WaterfallDialog GetWaterfallDialog()
+        {
+            var steps = new List<WaterfallStep>();
+
+            steps.AddRange(GenerateUpdateSteps<CaseManagementData>(Phrases.Capacity.CaseManagement.Service, nameof(CaseManagementData.Total),
+                nameof(CaseManagementData.Open), nameof(CaseManagementData.HasWaitlist), nameof(CaseManagementData.WaitlistLength),
+                Phrases.Capacity.CaseManagement.GetSpotsOpen));
```

Comment:
Each module now just adds the update steps for the fields it needs instead of the entire conversation flow